src/hooks/useTodoItemCardComplete.ts

```diff
@@ -1,19 +1,19 @@
 import axios, { AxiosError } from "axios";
 import { useState } from "react";
-import { ICard } from "../components/Card";
+import { ITodoItemCard } from "../components/todo-item-card";
 
-const useCardComplete = () => {
+const useTodoItemCardComplete = () => {
   const [isLoading, setIsLoading] = useState(false);
   const [error, setError] = useState("");
 
-  async function cardComplete(card: ICard) {
+  async function todoItemCardComplete(todoItemCard: ITodoItemCard) {
     try {
       setError("");
       setIsLoading(true);
-      card.isCompleted = true;
-      await axios.put<ICard>(
-        "https://aufgabenliste.azurewebsites.net/api/todo" + "/" + card.id + "?id=" + card.id,
-        card)
+      todoItemCard.isCompleted = true;
+      await axios.put<ITodoItemCard>(
+        "https://aufgabenliste.azurewebsites.net/api/todo" + "/" + todoItemCard.id + "?id=" + todoItemCard.id,
+        todoItemCard)
       setIsLoading(false);
     } catch (e: unknown) {
       const error = e as AxiosError;
```

Comment:
And, why do you need this new variable `error`? What are you doing with this?

Reply:
I have just use it from example. AndI guess this is for **Cast**.